ra_aid/agent_utils.py

```
@@ -181,6 +185,14 @@ def run_research_agent(
     code_snippets = _global_memory.get("code_snippets", "")
     related_files = _global_memory.get("related_files", "")
 
+    # Get project info
+    try:
+        project_info = get_project_info(".", file_limit=2000)
+        formatted_project_info = format_project_info(project_info)
+    except Exception as e:
```

Ruff (0.8.2)

Do not catch blind exception: `Exception`

(BLE001)